helm/values.yaml

```diff
@@ -0,0 +1,52 @@
+---
+replicaCount: 1
+
+securityContext:
+  fsGroup: 123
+
+container:
+  image: jmatejka/happy-bank-core
+  tag: 0.2.1
+  containerPort: 5000
+  volumeMountPath: /data
+  healthEndpoint: /health
+
+storageClass:
+  provisioner: kubernetes.io/no-provisioner
+  volumeBindingMode: WaitForFirstConsumer
+
+persistentVol:
+  accessMode: ReadWriteOnce
+  capacity: 1Gi
+  localPath: /data
+  node: minikube
+  reclaimPolicy: Retain
+  storageClass: local-storage
+
+persistentVolClaim:
+  accessMode: ReadWriteOnce
+  capacity: 1Gi
+  storageClass: local-storage
+
+service:
+  protocol: TCP
+  type: ClusterIP
```

Comment:
From my point of view, the service will always be running on TCP and set as the ClusterIP type.